lambda/custom/src/babybuddy/api.ts

```diff
@@ -13,10 +13,12 @@ import {
 
 const fetchSecrets: () => Promise<Secret> = async () => {
   return new Promise((resolve, reject) => {
-    if (process.env.BABY_BUDDY_API_KEY && process.env.BABY_BUDDY_API_URL) {
+    if (process.env.BABY_BUDDY_API_KEY && process.env.BABY_BUDDY_API_URL && process.env.CF_ACCESS_CLIENT_ID && process.env.CF_ACCESS_CLIENT_SECRET) {
       resolve({
         apiKey: process.env.BABY_BUDDY_API_KEY,
         apiUrl: process.env.BABY_BUDDY_API_URL,
+        cfAccessClientId: process.env.CF_ACCESS_CLIENT_ID,
+        cfAccessClientSecret: process.env.CF_ACCESS_CLIENT_SECRET
```

Comment:
Assuming the CF header variables can be made optional -- lets keep the current structure but but the object to go to `resolve`, add the CF headers if provided, and then call `resolve` at the end.